src/command_modules/azure-cli-appservice/azure/cli/command_modules/appservice/custom.py

```diff
@@ -335,13 +337,19 @@ def delete_container_settings(resource_group_name, name, slot=None):
 
 def show_container_settings(resource_group_name, name, slot=None):
     settings = get_app_settings(resource_group_name, name, slot)
-    return _filter_for_container_settings(settings)
+    return _mask_creds_related_appsettings(_filter_for_container_settings(settings))
 
 
 def _filter_for_container_settings(settings):
     return [x for x in settings if x['name'] in CONTAINER_APPSETTING_NAMES]
 
 
+def _mask_creds_related_appsettings(settings):
```

Comment:
Please add a TODO here to remove this workaround when the service side issue is fixed